hack/build_docs.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
do we need this?

Reply:
no, deleted this and all other unused files added in hack/